### ISSUE TITLE
[Bug Report] Title Here

### ISSUE THREAD

**Issue by logdotexe** [jagrosh/MusicBot#578]
**Describe the bug**
All commands works, but when I try to play songs or playlist, the bot joins and the leaving instantly. I search for similar bugs here but nothing of the solutions resolve my problem.

**To Reproduce**
1. send "!play song"
2. bot join
3. bot leave without playing anything
4. check console: no error

**Expected behavior**
works :)

**Screenshots**
https://ibb.co/hWRrxKW

**Debug Info**

```
System Properties:
  java.version = 11.0.9.1
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 11
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 11.0.9.1+1
  java.specification.version = 11
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.3.2
  Owner = 235476615965048832
  Prefix = !
  AltPrefix = null
  MaxSeconds = 0
  NPImages = false
  SongInStatus = false
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 4.2.0_214
  JDA-Utilities Version = 3.0.5
  Lavaplayer Version = 1.3.55

Runtime Information:
  Total Memory = 31
  Used Memory = 18

Discord Information:
  ID = 795819722313826305
  Guilds = 1
  Users = 2
```

**Additional context**
I run the bot in a docker container through Synology

**Comment by jagrosh** [comment]
There's not a lot to go on here, can you try each of the following (separately) to try to narrow down where the problem might be?
- change voice region in your Discord server
- use an older or oracle version of the jre
- run outside of a docker container

**Comment by logdotexe** [comment]
chage voice region not work
run outside of a docker container it works. I run it on my windows pc 
use and older or oracle version of the jre not work (i tried same as my pc: java 1.8.0_271)

**Comment by logdotexe** [comment]
I found the problem: Synology firewall. With that off I can play music, now I have to figure out which port I have to open.

**Comment by jagrosh** [comment]
The issue must be with your docker container or setup then. Unfortunately, that is out of the scope of this project unless you can provide any other info or logs.

**Comment by logdotexe** [comment]
Problem solved. For anyone with my problem in your Synology firewall you have to open port 443 TCP for textchat.  For voice, it is a random UDP port between 50000 - 65535

**Comment by Sanduhr32** [comment]
Yes of course you need the standard tcp port for the gateway open and UDP ports for voice.
